src/components/studies/StudyCard.tsx

```diff
@@ -104,17 +102,65 @@ const useStyles = makeStyles((theme: ThemeType) => ({
   studyCardTextField: {
     marginBottom: theme.spacing(2),
   },
+  isJustAdded: {
+    border: `3px solid ${theme.palette.primary.dark}`,
+    animation: '$pop-out 0.5s ease',
+  },
+  '@keyframes pop-out': {
+    '0%': {
+      transform: 'scale(0)',
+    },
+    '100%': {
+      transform: 'scale(1)',
+    },
+  },
 }))
 
 const cancelPropagation = (e: React.MouseEvent) => {
   e.stopPropagation()
   e.preventDefault()
 }
 
+const getFormattedDate = (date: Date) => {
```

Comment:
We  use momentjs library for date formatting 
https://momentjs.com/docs/#/displaying/